collie/model/hybrid_matrix_factorization.py

```diff
@@ -289,22 +289,17 @@ def _load_model_init_helper(self, load_model_path: str, map_location: str, **kwa
 
     def _configure_metadata_layers(
         self,
-        metadata_layers_attr: str,
-        metadata_layer_name: str,
-        metadata_layers_dims: Optional[Iterable],
+        metadata_type: str,
+        metadata_layers_dims: Optional[Iterable[int]],
```

Comment:
Can we add to the docstring of this method what attributes will be set as a result of calling this method? Specifically ``{metadata_type}_metadata_layers`` and ``{metadata_type}_metadata_layers_dims``.

Reply:
👍